### PR TITLE
Touch-action or Fastclick. Remove placeholder plugin

### PR DESCRIPTION
**Pull request by Gnito**
Cleaning some old codes.
- If touch-action is supported => No need for Fastclick
- Placeholder attribute is widely supported nowadays. Do we need extra js?
(i.e. removed https://github.com/mathiasbynens/jquery-placeholder )

### CODE REVIEW

**Comment by Gnito** [comment]
- We have quite few input fields that actually use placeholder texts. Most of them are in admin panel.
- I think there's no need to initialize FastClick library if there are touch-action put in place (so, I put it in place).
- I also thought of making FastClick as lazy loaded script, but I'm not sure if it's worth the trouble at this point.

**Comment by Gnito** [comment]
IE9 support for placeholder attributes is removed by removing this polyfill library. But then again, we are not supporting IE9 anymore.

File: app/views/layouts/application.haml [hunk 1]
```diff
@@ -67,8 +67,11 @@
 
   :javascript
     $(function() {
-      FastClick.attach(document.body, { excludeNode: '^pac-'} );
-      $('input, textarea').placeholder();
+      if ('touchAction' in document.body.style) {
+        document.body.style.touchAction = 'manipulation';
+      } else {
+        window.FastClick.attach(document.body, { excludeNode: '^pac-'} );
```
Comment (mporkola) [inline app/views/layouts/application.haml]: out of curiosity, what's `excludeNode: '^pac-'`? I see it's not new, but anyways.

Comment (Gnito) [inline app/views/layouts/application.haml]: FastClick & Google Autocomplete don't want to be in the same page. We had to patch FastClick while making Location search.

**Pull request merged**